app/services/search/book_versions/i1/indexing_strategy.rb

```diff
@@ -0,0 +1,57 @@
+module Search::BookVersions::I1
+  # The indexing strategy is the encapsulation for the index's structure and
+  # metdata (including index settings & mappings).
```

Comment:
I see the indexing strategy as mainly the strategy for _how_ to index.  The how is intertwined with the structure since Elasticsearch wants to know what the documents look like.